app/helpers/errors_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ErrorsHelper
+  def render_errors(form)
+    if form.object.errors.any?
```

Comment:
thoughts on assigning `form.object.errors` to a local variable to use in the conditional and render?